srcpkgs/persepolis/template

```diff
@@ -1,15 +1,21 @@
 # Template file for 'persepolis'
 pkgname=persepolis
-version=3.2.0
-revision=8
-build_style=python3-module
-hostmakedepends="python3-setuptools python3-setproctitle python3-requests
- python3-psutil aria2 ffmpeg libnotify pulseaudio-utils youtube-dl
- sound-theme-freedesktop python3-PyQt5-tools qt5-svg"
-depends="${hostmakedepends/python3-setuptools/}"
-short_desc="Download manager and GUI for Aria2"
+version=5.1.1
+revision=1
+build_style=meson
+build_helper=python3
+makedepends="python3-pysocks python3-urllib3 python3-pyside6 qt6-svg
+python3-requests python3-setproctitle meson python3-psutil ffmpeg yt-dlp
```

Comment:
why is meson needed as a makedepend? it's implied as a hostmakedepend by the build_style